### DISCUSSION TITLE
【低代码系列之AIGC】 人工智能会给低代码带来什么？

### DISCUSSION THREAD

**Discussion by giscus** [kinfuy/kinfuy.github.io#4] [General]
# 【低代码系列之AIGC】 人工智能会给低代码带来什么？

序体验了ChatGPT后，我还在感叹是不是人类要全面进入个人人工智能助手的时代了。然后就出现一系列的AI产品。  bing宣布整合ChatGPT(现在都还没有排上队)，bing 因此下载量暴涨，有现在已经把整合的功能阉割了，没有才开始那么智能了。    百度将在三月份推出生成式人工智能（AI）产品文心一言（ERNIE Bot），并宣布计划将多项主流业务与文心一言整合。股价都涨了一波。我体验了一下文

https://kinfuy.github.io/f117b534a1ee/

**Comment by kinfuy** [comment]
拥抱未来吧，时间啊🤔
